src/OSPSuite.UI/Views/QuantityListView.cs

```diff
@@ -207,36 +205,9 @@ public DataBrowserPopupMenu(GridViewBinder<QuantitySelectionDTO> gridViewBinder,
 
          private void updateSelection(bool selected)
          {
-            if (View.IsDataRow(_groupRowHandle)) return;
-            var selectedItems = selectedObjects(_groupRowHandle);
+            var selectedItems = _gridViewBinder.SelectedItems(_groupRowHandle);
             _presenter.UpdateSelection(selectedItems.ToList(), selected);
          }
-
-         //from DevExpressDocumentation XtraGrid Adv... Grouping "Process Group Rows"
-         private IEnumerable<QuantitySelectionDTO> selectedObjects(int groupRowHandle)
-         {
-            //Get the number of immediate children
-            if (!View.IsGroupRow(groupRowHandle))
-               yield break;
-
```

Comment:
That same code was not once, or twice but 3 TIMES! And twice for the same viiew. Crazy